.rubocop.yml

```diff
@@ -1158,4 +1158,4 @@ Style/WordArray:
   Enabled: false
 
 Style/FrozenStringLiteralComment:
-  Enabled: true
+  Enabled: false
```

Comment:
@jamaga This was explicitly enabled before, did @mikz or @hallelujah agreed to change it?